packages/phoenix-ng/projects/phoenix-app/cypress/e2e/index.test.ts

```diff
@@ -0,0 +1,47 @@
+describe('Test the index page of Phoenix', () => {
+  it('should match the whole page', () => {
+    cy.visit('/').then(() => {
+      cy.document().toMatchImageSnapshot();
+    });
+  });
+
+  it('should navigate to the correct page', () => {
+    // there will only be one link within the card
+    cy.get('.card a').eq(0).click();
+    cy.url().should('eq', Cypress.config().baseUrl + 'playground');
+    cy.go('back');
+    cy.get('.card a').eq(1).click();
+    cy.url().should('eq', Cypress.config().baseUrl + 'geometry');
+    cy.go('back');
+    cy.get('.card a').eq(2).click();
+    cy.url().should('eq', Cypress.config().baseUrl + 'atlas');
+    cy.go('back');
+    cy.get('.card a').eq(3).click();
+    cy.url().should('eq', Cypress.config().baseUrl + 'lhcb');
+    cy.go('back');
+    cy.get('.card a').eq(4).click();
+    cy.url().should('eq', Cypress.config().baseUrl + 'cms');
+    cy.go('back');
+    cy.get('.card a').eq(5).click();
+    cy.url().should('eq', Cypress.config().baseUrl + 'trackml');
+    cy.go('back');
+  });
+
+  it('should point to correct links in footer', () => {
```

Comment:
I can't classify which type of test this would be. I don't think we need it. I would recommend removing it.